instances/treasury-factory.near/widget/components/create-treasury/ConfirmWalletStep.jsx

```diff
@@ -0,0 +1,115 @@
+const { getNearBalances } = VM.require(
+  "${REPL_DEVDAO_ACCOUNT}/widget/lib.common"
+);
+
+const baseUrl = "https://api.pikespeak.ai";
+const [balance, setBalance] = useState(null);
+const REQUIRED_BALANCE = 12;
+
+const getBalance = () => {
+  const balance = getNearBalances(context.accountId);
+  setBalance(parseFloat(balance.availableParsed));
+};
+
+getBalance();
+
```

Comment:
should we do it in useEffect in first render? to avoid calling it multiple times with change in state

Reply:
It containis Near `fetch` fn that is async. In case of first re-render it returns null.
using new Promise(...) doesn't work well here.
Also I've seen iin the codebase that this is common approach to get user balance.

Here we can omiit states and use variable (as in codebase). but it's not a true react way